core/iwasm/common/wasm_runtime_common.c

```diff
@@ -2536,9 +2730,9 @@ static VoidFuncPtr invokeNative_Void = (VoidFuncPtr)(uintptr_t)invokeNative;
 
 bool
 wasm_runtime_invoke_native(WASMExecEnv *exec_env, void *func_ptr,
-                           const WASMType *func_type, const char *signature,
-                           void *attachment, uint32 *argv, uint32 argc,
-                           uint32 *argv_ret)
+                           bool is_aot_func, const WASMType *func_type,
+                           const char *signature, void *attachment,
+                           uint32 *argv, uint32 argc, uint32 *argv_ret)
```

Comment:
Should also handle EXTERNREF with wasm_externref_ref2obj in this function? see L2942

Reply:
It seems there is no `funcref` and `externref` support when `defined(BUILD_TARGET_ARM_VFP) || defined(BUILD_TARGET_THUMB_VFP)`. Shall we enable them ?

Reply:
Yes, normally we should support it in all archs.